TTS/auto_tts/single_speaker_autotts.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+
+from TTS.auto_tts.complete_recipes import TtsAutoTrainer
+
+
+def main():
+    parser = argparse.ArgumentParser()
```

Comment:
For Argparsing over default values you can use Coqpit https://github.com/coqui-ai/coqpit